okteflix/templates/ingress.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ include "okteflix.fullname" . }}
+  labels:
+    {{- include "okteflix.labels" . | nindent 4 }}
+  {{- with .Values.ingress.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  tls:
+  - hosts:
+    - dev-okteflix-ramiro.sf.okteto.net
```

Comment:
This needs https://github.com/okteto/app/pull/519 so the host doesn't need to be specified